packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -84,7 +80,6 @@ export interface IConnectableRuntime {
 	readonly disposed: boolean;
 	readonly connected: boolean;
 	readonly clientId: string | undefined;
-	readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
```

Comment:
Why are you changing who holds the DeltaManager?  Is it needed in more places that can't access this?

Reply:
I guess it's from the observation that RunningSummarizer used to have `ISummarizerRuntime` but only ever access `deltaManager` on it - so just cut out the middle man?

Reply:
With this change, the deltamanager on container runtime is a proxy but the summarizer classes need the real delta manager mostly because they check the `active` method on delta manager.
If these classes directly accessed container runtime's delta manager, they would get the proxy. So, I removed the direct access and passed in the real delta manager directly.

Reply:
Looks like it was mostly reverted - why not go all the way back?

Reply:
Oh I see, never mind.  Looks good to lock it down a bit more since it's no longer used - you're passing just the `isActive` callback to that code now instead.